src/get-started/java-binding.md

```diff
@@ -103,6 +102,36 @@ blockchain state hash, which is included in each committed block. When using
 implements [`Schema`][schema] interface; when implementing
 `Service` directly – in the service itself.
 
+#### Serialization
+
+As Exonum storage accepts data in the form of byte arrays,
+storing user data requires serialization.
+Java Binding provides a set of built-in *serializers* that you can find
+at the [`StandardSerializers`][standardserializers] utility class.
+The list of serializers covers the most often-used entities and includes:
+
+- Standard types: `boolean`, `float`, `double`, `byte[]` and `String`.
+  Integers with various encoding types,
+  see [`StandardSerializers`][standardserializers] Java documentation
+  and the table below.
+- Exonum types: `PrivateKey`, `PublicKey` and `HashCode`.
+- Any Protobuf messages using `StandardSerializers#protobuf`.
+
+Besides the available built-in serializers, users can still implement
+their own serializers for storing their data in a custom format instead
+of using the built-in ones.
+
+##### Integer Encoding Types Comparison Table
```

Comment:
@slowli , @Maria-Nosyk is there such a thing in this markdown as a table title? Or shall one resort to HTML if needs to specify the title?